.circleci/build.sh

```diff
@@ -14,11 +14,20 @@ sed -i -e "s/##//" docker-compose.yml
 # Pull the latest images.
 ahoy pull
 
-# Disable checks used on host machine.
-export DOCTOR_CHECK_PYGMY=0
-export DOCTOR_CHECK_PORT=0
-export DOCTOR_CHECK_SSH=0
-export DOCTOR_CHECK_WEBSERVER=0
-export DOCTOR_CHECK_BOOTSTRAP=0
+if [ "$CKAN_VERSION" = "2.8" ]; then
```

Comment:
please include in README.md how to run locally by running this script .

Reply:
Done.